conf/establishersCompany.routes

```diff
@@ -262,8 +264,10 @@ POST       /:index/company/what-you-will-need
 GET        /:index/company/:mode/what-you-will-need/:srn                                                    controllers.register.establishers.company.WhatYouWillNeedCompanyDetailsController.onPageLoad(mode: Mode, srn: Option[String], index: Index)
 POST       /:index/company/:mode/what-you-will-need/:srn                                                    controllers.register.establishers.company.WhatYouWillNeedCompanyDetailsController.onSubmit(mode: Mode, srn: Option[String], index: Index)
 
-GET        /:index/company/company-number                                                                   controllers.register.establishers.company.HasCompanyNumberController.onPageLoad(mode: Mode = NormalMode, srn: Option[String] = None, index: Index)
```

Comment:
Do we need to get rid of the old URL?

Reply:
its not old, its new was newly added which got updated with prefix has-